FTBFS musl

```
_build-podman-xbps/src/github.com/containers/libpod/pkg/rootless/rootless_linux.go:27:8: error: unknown type name 'uid_t'
 extern uid_t rootless_uid();
        ^~~~~
_build-podman-xbps/src/github.com/containers/libpod/pkg/rootless/rootless_linux.go:28:8: error: unknown type name 'uid_t'
 extern uid_t rootless_gid();
        ^~~~~
```

--- a/pkg/rootless/rootless_linux.go
+++ b/pkg/rootless/rootless_linux.go
@@ -24,6 +24,7 @@ import (
 /*
 #cgo remoteclient CFLAGS: -DDISABLE_JOIN_SHORTCUT
 #include <stdlib.h>
+#include <sys/types.h>
 extern uid_t rootless_uid();
 extern uid_t rootless_gid();
 extern int reexec_in_user_namespace(int ready, char *pause_pid_file_path, char *file_to_read, int fd);

